tests/integration/cv_aggregator.cpp

```diff
@@ -0,0 +1,112 @@
+#include <iostream>
+
+#include <opencv2/opencv.hpp>
+#include <loguru.hpp>
+
+#include "cv/pipeline.hpp"
+#include "cv/aggregator.hpp"
+
+// Download these test images from one of the zips here https://drive.google.com/drive/u/1/folders/1opXBWx6bBF7J3-JSFtrhfFIkYis9qhGR
+// Or, any cropped not-stolen images will work
+
+// this image should be located at a relative path to the CMake build dir
+const std::string imageTestDir = "../tests/integration/images/matching_cropped/test/";
+const std::string refImagePath0 = imageTestDir + "000000910.jpg"; // bottle 4
+const std::string refImagePath1 = imageTestDir + "000000920.jpg"; // bottle 3
+const std::string refImagePath2 = imageTestDir + "000000003.jpg"; // bottle 2
+const std::string refImagePath3 = imageTestDir + "000000004.jpg"; // bottle 1
+const std::string refImagePath4 = imageTestDir + "000000005.jpg"; // bottle 0
+
+// matching model can be downloaded from here: https://drive.google.com/drive/folders/1ciDfycNyJiLvRhJhwQZoeKH7vgV6dGHJ?usp=drive_link
```

Comment:
uh..
> and we aren't exposing any of the real trade secrets (CV/ML models)

Reply:
uhhhhhhhhhhh

Reply:
should we try and obfuscate this @atar13